docs/user-guide/configuration/externalConfig.md

```diff
@@ -49,11 +51,31 @@ jobs:
 
 Builds for child pipeline will have access to parent pipeline's repository at [`$SD_CONFIG_DIR`](../environment-variables#directories).
 
+## Manage Child Pipelines
+When a child pipeline SCM URL is added to the parent pipeline configuration, child pipeline gets created with `ACTIVE` state.
+
+Child pipeline state will be set to `INACTIVE` when the associated SCM URL has been removed from the parent pipeline configuration.
```

Comment:
should we mention, parent pipeline configuration in the screwdriver.yaml ?

Reply:
Done